src/solvers/refinement/string_refinement.cpp

```diff
@@ -1533,3 +1618,33 @@ exprt string_refinementt::get(const exprt &expr) const
 
   return substitute_array_lists(ecopy);
 }
+
+/// Creates a solver with `axiom` as the only formula added and runs it. If it
+/// is SAT, then true is returned and the given evalutation of `var` is stored
+/// in `witness`. If UNSAT, then what witness is is undefined.
+/// \param [in] axiom: the axiom to be checked
+/// \param [in] var: the variable whose evaluation will be stored in witness
+/// \param [out] witness: the witness of the satisfying assignment if one
+///   exists. If UNSAT, then behaviour is undefined.
+/// \return: true if axiom is SAT, false if UNSAT
+bool string_refinementt::is_axiom_sat(
```

Comment:
Wait, what's bad about it? It's supposed to be 2 spaces from the line above right? That's how I've seen other comments (also others in this file).